ptable/ptable.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Partition table
+ *
+ * Copyright 2020 Phoenix Systems
+ * Author: Hubert Buczynski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+
+#ifndef _LIB_PARTITION_TABLE_H_
+#define _LIB_PARTITION_TABLE_H_
+
+
+/* Structure of partition table:
+*  _________________________________________________________________________________
+* |        28 B     |                      32 B * n                     |    4 B    |
+*  ---------------------------------------------------------------------------------
+* | ptable_header_t | ptable_partition_t 0 | ... | ptable_partition_t n | magicBytes|
+*  ---------------------------------------------------------------------------------
+*/
+
+
+static const uint8_t magicBytes[] = {0xde, 0xad, 0xfc, 0xbe};
```

Comment:
actually exporting symbol `magicBytes` may break some programs which would like to use this header. Add prefix.